### DISCUSSION TITLE
how to get first error from createItem call - asked again

### DISCUSSION THREAD

**Discussion by box-turtle** [microsoft/Microsoft-Fabric-workload-development-sample#136] [Help]
@ramizsha  Posting a repeat of a question already marked as `Answered` because we can't get the answer to work.

Please see [my comment here](https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/discussions/43#discussioncomment-11019771) .

1. We still don't see our back end error payload propagated to front end by Fabric in the dev tools
2. We don't get the first error thrown in the `catch` block of `workloadClient.itemCrud.createItem` - only the error from the retry attempt initiated not by us but by Fabric client that gives the misleading `PowerBIMetadataArtifactDisplayNameInUseException`

**Comment by ramizsha** [comment]
Marked as unanswered on the other discussion, closing this one.
